application/common/mail/welcome.html.php

```diff
@@ -0,0 +1,86 @@
+<?php
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $employeeId
+ * @var string $firstName
+ * @var string $lastName
+ * @var string $displayName
+ * @var string $username
+ * @var string $email
+ * @var string $active
+ * @var string $locked
+ * @var string $lastChangedUtc
+ * @var string $lastSyncedUtc
+ * @var string $lastLoginUtc
+ * @var string $passwordExpiresUtc
+ * @var string $emailSignature
+ * @var string $helpCenterUrl
+ * @var string $idpDisplayName
+ * @var string $passwordForgotUrl
+ * @var string $passwordProfileUrl
+ * @var string $supportEmail
+ * @var string $supportName
+ * @var bool   $isMfaEnabled
+ */
+?>
+<p>
+    Dear <?=yHtml::encode($displayName)?>,
+</p>
+<p>
+    Congratulations! You have logged into your new <?=yHtml::encode($idpDisplayName)?> account for the first time.
```

Comment:
Instead of "account" I wonder if we could come up with a clearer wording, e.g. "online account" or "web account". (Applies to the text email message too.)